scripts/copydocs.py

```diff
@@ -8,9 +8,9 @@
 
 import web
 
-from scripts.solr_builder.solr_builder.fn_to_cli import FnToCLI
-
 sys.path.insert(0, ".")  # Enable scripts/copydocs.py to be run.
+
```

Comment:
```suggestion
import _init_path  # noqa: E402,F401
```
With this change, we can also remove `sys.path.insert(0, ".")` (line 11), and the `import scripts._init_path` line (line 14).

Reply:
I made that change, but it seems to cause the python test to fail.